spec/asyncapi.md

```diff
@@ -554,142 +553,147 @@ Field Pattern | Type | Description
 ```
 
 ```yaml
-user/signedup:
-  subscribe:
-    message:
-      $ref: "#/components/messages/userSignedUp"
+userSignedUp:
+  address: 'user.signedup'
+  messages:
+    userSignedUp:
+      $ref: '#/components/messages/userSignedUp'
 ```
 
 
 
 
-#### <a name="channelItemObject"></a>Channel Item Object
+#### <a name="channelObject"></a>Channel Object
 
-Describes the operations available on a single channel.
+Describes a shared communication channel.
```

Comment:
A shared communication channel with who? Am I the only one who feels like this sentence is incomplete in some way... 🤔

Reply:
We don't know in advance. We only know it's gonna be shared 🤷